captainhook/checkers/yamllint_checker.py

```diff
@@ -0,0 +1,42 @@
+# # # # # # # # # # # # # #
+# CAPTAINHOOK IDENTIFIER  #
+# # # # # # # # # # # # # #
+import os
+from .utils import bash, get_config_file
+
+DEFAULT = 'off'
+CHECK_NAME = 'yamllint'
+NO_YAMLLINT_MSG = ("yamllint is required for the yamllint plugin.\n"
+                "`pip install yamllint` or turn it off in your {} file.".format(get_config_file()))
+REQUIRED_FILES = ['.yamllint']
+
+
+def _filter_yaml_files(files):
```

Comment:
The comments indicate that this was copy/pasted; should we instead have this in `.utils` so both the Python, YAML and future checkers can use the same code path?